README.md

```
@@ -64,6 +64,20 @@ pnpm lighthouse:desktop - lighthouse desktop testing
 pnpm lighthouse:mobile - lighthouse mobile tesitng
 ```
 
+## Routing
+
+This project includes a routing script for managing URLs. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)